app/controllers/api/v1/people_controller.rb

```diff
@@ -36,6 +36,15 @@ def image
 
     private
 
+      def import_person_from_nomis prison_number
+        # This prevents us from blaming the current user/application for the NOMIS sync
+        PaperTrail.request(whodunnit: nil) do
+          Moves::ImportPeople.new([person_nomis_prison_number: prison_number]).call
+        end
+      rescue StandardError => e
```

Comment:
maybe catch a more specific error, rather than everything ?

Reply:
Its deliberate. It's a pattern that was in the other synchroniser - whatever goes wrong during sync, we don't want the application to fail, but we do want it reported to sentry